README.md

```diff
@@ -19,9 +19,9 @@ secrets and config maps available in multiple namespaces.
 
 ### Using Helm
 
-```shellsession
-$ helm upgrade --install kubernetes-replicator ./deploy/helm-chart/kubernetes-replicator
-```
+1. [Add the Mittwald-Charts Repo](https://github.com/mittwald/helm-charts/blob/master/README.md#usage)
```

Comment:
Maybe just so you don't have to switch pages to quickly setup the chart:

```suggestion
1. [Add the Mittwald-Charts Repo](https://github.com/mittwald/helm-charts/blob/master/README.md#usage):

    ```
    $ helm repo add mittwald https://helm.mittwald.de && helm repo update
    ```